news/126.bugfix

```diff
@@ -0,0 +1,2 @@
+Use document_view as default for site root.
```

Comment:
@agitator @jensens this changelog entry makes it sound like it would change behavior. Not just a test. Technically this is also not a bugfix but an "internal" change I'd say.